src/language/en-GB/en-GB.plg_api_users.ini

```diff
@@ -3,6 +3,8 @@ PLG_API_USERS_DESCRIPTION="This plugin exposes users to the Joomla! API. Support
 PLG_API_USERS_BAD_REQUEST_MESSAGE="Bad request"
 PLG_API_USERS_REQUIRED_DATA_EMPTY_MESSAGE="Required data is empty"
 PLG_API_USERS_ACCOUNT_CREATED_SUCCESSFULLY_MESSAGE="Congratulations! Your account has been created successfully"
+PLG_API_USERS_ACCOUNT_CREATED_SUCCESSFULLY_MESSAGE_WITH_ACTIVATION_LINK="Congratulations! Your account has been created successfully. Verify your email to activate your account"
```

Comment:
`Congratulations! Your account was successfully created. Please verify your e-mail address to activate your account.`